exercises/managing-dns/overview.md

```diff
@@ -36,3 +144,5 @@ Specify a nameserver to check using @
 - How do our local machines know which DNS server to communicate with?
 - How can we find out what DNS records exist on a domain?
 - How do you manage DNS entries as terraform resources?
```

Comment:
Yeah I think not especially since we talk about API access being essential to doing this.